### DISCUSSION TITLE
The Current Input Timing is not Supported by the Monitor Display

### DISCUSSION THREAD

**Discussion by Akshay-Hegde** [ltsp/ltsp#434] [General]
I am having strange issue, I can boot on normal desktop as well as on virtualbox as diskless client without issue, but if I try to boot through thinclient ( TS100 ) receiving below error on display just before login screen, I tried changing multiple display too. 

processor : AMD
memory : 2GB

Is there any resolution for below issue ? 

![error-message-4](https://user-images.githubusercontent.com/7464492/113475989-15ff7300-9496-11eb-8f2f-5c0932f5cab6.jpg)

```
# ltsp info
LTSP version: 21.01-1~ubuntu21.04.1

SERVER XSESSIONS: lightdm-xsession.desktop  xfce.desktop

SERVER OS: PRETTY_NAME="Debian GNU/Linux 10 (buster)"

CHROOTS:


VMs:


IMAGES:
x86_64
```
![t3](https://user-images.githubusercontent.com/7464492/113476144-34b23980-9497-11eb-8406-85c2ab63b1c5.jpeg)
![t2](https://user-images.githubusercontent.com/7464492/113476146-35e36680-9497-11eb-893e-6165146e6803.jpeg)
![t1](https://user-images.githubusercontent.com/7464492/113476148-367bfd00-9497-11eb-9fbf-cbd26f00177b.jpeg)

**Comment by alkisg** [comment]
This sounds like a Xorg issue, which is unrelated to LTSP.
Try to boot the client with a live USB stick.
If it has the same problem, then you may contact the Ubuntu/Xorg communities for help.

You may also use epoptes (https://ltsp.org/docs/installation/) to see if you can monitor the client screen via VNC, and run `xrandr` commands to see if you can set an appropriate mode.
